templates/citizen_project/edit.html.twig

```diff
@@ -6,11 +6,14 @@
     {% import 'javascript.js.twig' as js %}
     <script src="{{ asset('ckeditor/ckeditor.js') }}"></script>
     <script src="{{ asset('ckeditor/lang/fr.js') }}"></script>
+    <script type="text/javascript" src={{ asset('bundles/sonataadmin/vendor/jquery/dist/jquery.min.js') }}></script>
+    <script type="text/javascript" src={{ asset('bundles/sonataadmin/vendor/jqueryui/ui/jquery-ui.js') }}></script>
     <script type="text/javascript">
         Kernel.onLoad(function() {
             {{ js.address(form.address) }}
             {{ js.ckeditor('citizen_project_proposed_solution') }}
             App.runCitizenProjectCreation();
+            {{ js.citizenProjectSkills('#citizen_project_skills', '#citizen_project_skills_search', path('app_citizen_project_skills_autocomplete')) }}
```

Comment:
I don't understand why we need a macro for this.
We could achieve the same behavior with a Javascript file.

Reply:
because in history other autocomplete JS functionality was wrote in macro, I follow the history...

Reply:
I would say that it's never too late to change our habits :P. 
Perhaps it was done for a very good reason in the 1st place ? For this case I don't see any reason why we should do like this.
Anyway I think we can keep it like this but it would be cool considering not to use macro by default in the future.